overlay.py

```diff
@@ -81,32 +85,26 @@ def publish_recording_status(self, message: str) -> None:
 	def connect(self, ip="192.168.100.100", port=1883):
 		self.client.connect_async(ip, port, 60)
 
-		try:
-			with BackendFactory.create(self.backend_name, self.width, self.height, self.publish_recording_status, self.publish_errors) as self.backend:
+		with BackendFactory.create(self.backend_name, self.width, self.height, self.publish_recording_status, self.publish_errors) as self.backend:
```

Comment:
We should probably still have some sort of error handling on the initialisation of the backend.